draft-ion.adoc

```diff
@@ -52,50 +58,59 @@ However, JSON's simple grammar can support virtually any data model - additional
 context is required for machine clients and servers to interpret data and
 metadata in important ways.
 
-Ion exists to codify specific name/value pairs within a JSON object to better
-facilitate JSON-based data and metadata exchange between machine clients and
-servers. As such, Ion does not define a new data grammar - it is a
-Media Subtype of `application/json`.
+Ion codifies specific name/value pairs within JSON objects to better
+facilitate JSON-based data and metadata exchange between machines.  And while machine interchange is the primary goal, Ion's second most important design goal is to enable intuitive and context-relevant structure so Ion content is very easy to understand for humans, even if they do not know what Ion is.  As such, Ion does not define a new data grammar - it is a Media Subtype of `application/json`.
 
 === Terminology
 
 These terms are defined by this specification:
 
+.Member
+A JSON name/value pair as defined in https://tools.ietf.org/html/rfc7159#section-4[RFC 7159].
+
 .Metadata
-Data about a resource.
+A set of data that provides information about other data.
 
 .Meta Object
-A JSON object that contains resource metadata.
+A JSON object with one or more members that all represent Metadata.
+
+.Root Object
+The single JSON object at the root of an Ion content structure.
```

Comment:
Yes, but there might have been some confusion here.

The Terminology section (1.1) defines terms for understanding while reading the document (so I'm trying to keep them clear and concise and not overwhelming), but do not enumerate all the requirements for each term.

Section 2 is the normative section that addresses your concern and it does use the `MUST` notational convention.